apis/conversion/src/hypercube/__tests__/import-properties.spec.js

```diff
@@ -99,5 +105,7 @@ describe('importProperties', () => {
     expect(helpers.copyPropertyIfExist.callCount).to.equal(2);
     expect(helpers.copyPropertyOrSetDefault.callCount).to.equal(6);
     expect(helpers.importCommonProperties.callCount).to.equal(1);
+    expect(helpers.updateDimensionsOnAdded.callCount).to.equal(1);
+    expect(helpers.updateMeasuresOnAdded.callCount).to.equal(1);
```

Comment:
The tests here only check if they've been called, which doesn't really tell us anything.
Should write a separate test to run teh functionality of the updateXYZOnAdded functions.

Reply:
Will update it!

Reply:
Updated!